src/nft/hooks/usePriceImpact.ts

```diff
@@ -0,0 +1,46 @@
+import { Currency, Percent, TradeType } from '@uniswap/sdk-core'
+import { useMemo } from 'react'
+import { InterfaceTrade } from 'state/routing/types'
+import { useTheme } from 'styled-components/macro'
+import { computeRealizedPriceImpact, getPriceImpactWarning } from 'utils/prices'
+
+export interface PriceImpact {
+  percent: Percent
+  warning?: 'warning' | 'error'
+  toString(): string
```

Comment:
instead of overwriting the built in toString I would rename this `displayPercentage`. When I first saw `toString` I assumed it would also display all params such as the warning level and color